### ISSUE TITLE
Mimosa refresh behavior

### ISSUE THREAD

**Issue by neverfox** [dbashford/mimosa#199]
Currently, `mimosa refresh` deletes only the node modules that mimosa requires (express, consolidate, etc.) and then it removes their entries from `package.json` and rewrites the other dependencies you've added in a seemingly random order. It also ignores the versions listed in package.json during the reinstall (e.g. I had express at ~3.2.5 and refresh installed 3.2.4). Would it not work to have it simply remove all packages listed in package.json (even custom ones) and reinstall them using the version ranges you've specified there, not rewriting package.json in the process (since I like to keep my packages listed in alphabetical order).

In other words, is there a reason that refresh cannot just be a shortcut for:
- delete node_modules (or the contents)
- run `npm install`

Maybe a flag would serve the purpose, so the original behavior of refresh stays as it is, something like `mimosa refresh --custom` or the like. Using that flag would utilize the `package.json` found in the project as is.
